### DISCUSSION TITLE
Last Day Last Retweet problem!

### DISCUSSION THREAD

**Discussion by alikiraan1** [IBAX-io/go-ibax#1721] [Bug Submission]
Hi, 

Not accepting the last 5000 points retweet post. I tried many times. 
it's last day of airdrop. Is IBAX going to fix it till end of day?

Thank you
Ali

**Comment by alisabanoglu** [comment]
bir saattir cep telefonundan , bilgisayardan deniyorum hata veriyor:((((

**Comment by Abdullahoda** [comment]
açıklama yapıldı yoğunluktan dolayı verilecek merak etmeyin
